pom.xml

```diff
@@ -37,6 +37,18 @@
         </license>
     </licenses>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.jenkins.tools.bom</groupId>
+                <artifactId>bom-2.289.x</artifactId>
```

Comment:
Hopefully this means that next time we bump the minimum Jenkins version we don't have to also find out all the associated dependencies that need to be bumped.